rust/libraries/acars_config/src/sanity_checker.rs

```diff
@@ -6,140 +6,139 @@
 //
 
 // File to verify the sanity of config options
+// We don't have to verify the log level because the config_options module
+// Sanitizes that to a sane value
 
-use crate::config_options::Input;
+// Also, any input option set to be *only* numeric/u64 (ie, skew_window)
+// Will always be a valid number because the Clap parser will die if the input is bad
+
+use crate::Input;
 use std::net::SocketAddr;
 use std::str::FromStr;
 
 pub fn check_config_option_sanity(config_options: &Input) -> Result<(), String> {
     let mut is_input_sane = true;
-
-    // We don't have to verify the log level because the config_options module
-    // Sanitizes that to a sane value
-
-    // Also, any input option set to be *only* numeric/u64 (ie, skew_window)
-    // Will always be a valid number because the Clap parser will die if the input is bad
-
+    
     if !check_ports_are_valid(
```

Comment:
For a future update.

This section of code looks like a lot of the same sort of thing. would it be possible to create a static array/map with the desired options and iterate over it? this would lead to shorter code, fewer bug chances and easier to reason about.